lighthouse-core/lib/locales/index.js

```diff
@@ -3,11 +3,17 @@
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
-// @ts-nocheck
 'use strict';
 
-module.exports = {
+/** @typedef {Record<string, {message: string}>} LocaleMessages */
+
+/** @type {Record<LH.Locale, LocaleMessages>} */
+const locales = {
+  'ar': require('./ar-XB.json'), // TODO: fallback not needed when ar translation available
+  'ar-XB': require('./ar-XB.json'),
+  'en': require('./en-US.json'), // en-* fallback
```

Comment:
both `ar` and `en` will exist when we get our full languages back. If we want to do other mappings we can do them in here like this or make a simple map in `i18n.js` internal to `lookupLocale()`